src/Language/CSharp.ts

```diff
@@ -36,6 +36,12 @@ const unicode = require("unicode-properties");
 type Version = 5 | 6;
 type OutputFeatures = { helpers: boolean; attributes: boolean };
 
+enum AccessModifier {
+    None,
+    Public,
+    Internal
+}
+
```

Comment:
Would you mind splitting this up into two commits? One with the access modifiers, the other one with the `DateTimeOffset`?  It makes it much easier to understand what's going on and track down bugs if commits are one per feature or fix.